### ISSUE TITLE
Add DLL product data

### ISSUE THREAD

**Issue by jonpas** [intercept/intercept#147]
http://i.imgur.com/NQZhlKm.png

https://github.com/acemod/ACE3/blob/master/extensions/common/ace_version_win32.rc.in

**Comment by dedmen** [comment]
Windows only afaik.
If we can tell CMake how to do that it's easy to add. Though we need some way to grab the current version. Which CI could easilly do. See https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/ts/src/version.h#L3
Appveyor set's env variable `set AppVeyor=AppVeyorBuild="%APPVEYOR_BUILD_VERSION%";`
and the MSVC project has `$(AppVeyor)` in predefined macros. Which get's replaced by the env variable.

**Comment by jonpas** [comment]
Yes, Windows only. ACE3 does it [here](https://github.com/acemod/ACE3/blob/master/extensions/CMakeLists.txt#L102-L113)

**Comment by jonpas** [comment]
Oh yeah, forgot to note, this fixes `Unable to get file version size:` RPT message.

**Comment by jonpas** [comment]
Also add [`RVExtensionVersion`](https://github.com/acemod/ACE3/blob/73edbed1dc091d9ba7cc0aac72cbbe521715cd70/extensions/advanced_ballistics/AdvancedBallistics.cpp#L342-L349)

**Comment by jonpas** [comment]
This is best done after we get version checking in, as that will provide the required defines for use in both `RVExtensionVersion` and Windows file details.

**Comment by dedmen** [comment]
I'd like to have this before the next pre-release. As this is also important when analyzing crash dumps.
